theme/scriptsearch.less

```diff
@@ -93,4 +93,8 @@
     .ui.card.file .content {
         margin-right: 3rem;
     }
+}
+
+.dropdown-menu {
```

Comment:
theoretically this https://github.com/microsoft/pxt/blob/master/theme/themes/pxt/modules/dropdown.overrides#L53 should be setting the z-index, not sure if something changed or there was a missed case

Reply:
Yep, my css must have been super out of date. removed from pr